### ISSUE TITLE
reduce() is deprecated in Python3

### ISSUE THREAD

**Issue by alertedsnake** [elfsternberg/git-linter#1]
According to [Python3's "whats new" page](https://docs.python.org/3.0/whatsnew/3.0.html#builtins), reduce() is deprecated, and one should either use a for loop, or if absolutely necessary, functools.reduce().  As it is, this is broken on 3.

It's used here:
[git_lint/reporters.py#L35](https://github.com/elfsternberg/git-linter/blob/f70aea7ec337a20926b5e214f70602d8cc70bf69/git_lint/reporters.py#L35)

The actual error:

```
Traceback (most recent call last):
  File "/usr/bin/git-lint", line 9, in <module>
    load_entry_point('git-linter==0.0.6', 'console_scripts', 'git-lint')()
  File "/usr/lib/python3.5/site-packages/git_lint/__main__.py", line 57, in main options)
  File "/usr/lib/python3.5/site-packages/git_lint/reporters.py", line 36, in print_report
    messages = reduce(operator.add, [item[3] for item in group[1]], [])
NameError: name 'reduce' is not defined
```

**Comment by elfsternberg** [comment]
Thank you.  Found and fixed.

**Comment by alertedsnake** [comment]
Awesome!

**Comment by elfsternberg** [comment]
Pushed to pypi, fixed in 0.0.7.
